### PR TITLE
Use custom notify_push message to update page list

### PR DESCRIPTION
**Pull request by mejo-**
### 📝 Summary

Ensures that page list is refreshed on each relevant change.

At the same time makes sure it's only refreshed on page list changes, not on other filesystem changes.

Fixes: #441

Signed-off-by: Jonas <jonas@freesources.org>

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits

### CODE REVIEW

**Review by juliushaertl — APPROVED** [review]
Looks good but haven't tested.

**Comment by juliushaertl** [comment]
Seems the commit from #526 slipped in here as well ;)

File: lib/Service/PageService.php [hunk 1]
```diff
@@ -27,6 +29,8 @@ class PageService {
 	private UserFolderHelper $userFolderHelper;
 	private IUserManager $userManager;
 	private IConfig $config;
+	private ContainerInterface $container;
+	private $pushQueue;
```
[the remark this answers is not in the document]
Comment (mejo-) [reply lib/Service/PageService.php:33]: Not sure, but will give it a try. It needs to work without notify_push being installed.

[the remark this answers is not in the document]
Comment (mejo-) [reply lib/Service/PageService.php]: Nice, with `private ?IQueue = null;` it works.

**Pull request merged**